docs/guidelines/content/subsets.md

```diff
@@ -113,7 +113,7 @@ The approval process and claiming of a subset will vary depending on if the subs
 
 ² To Contact Developer Compliance, send them a site message [here](https://retroachievements.org/createmessage.php?t=DevCompliance&s=Subset%20Approval%20Request%20-%20[Game%20Name]%20-%20[Subset%20Name]) and replace the `[Game Name]` and `[Subset Name]` in the subject line with the game name and desired subset name.
 
-³ _The following are pre-approved subset types and therefore do not require contacting Developer Compliance:_<br>
+³ The following are pre-approved subset types and therefore do not require ³ _The following are pre-approved subset types and therefore do not require contacting Developer Compliance:_<br>
```

Comment:
![Screenshot 2024-09-01 at 1 28 54 PM](https://github.com/user-attachments/assets/19d1c94f-56a0-4e71-ab87-523e8a222ec9)

This should be rolled back.